core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -131,6 +191,32 @@ var DiagnosticTestPage = function () {
       expect(recommendedTopicSummaryTiles.length).toEqual(0);
     }
   };
+
+  var AutocompleteTopicDropdownEditor = async function (topicName) {
```

Comment:
Since this is a function and not a class, please use `camelCase` naming instead of `CamelCase`

Reply:
Done.